tools/dockerfile-add-scanner/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Use an official Golang runtime as a parent image
+FROM golang:latest as builder
```

Comment:
Can we pin this to 1.20.1 which is what we use elsewhere?
And ideally use some ENV variable called GOVER to make it easier to find when we want to update?

Reply:
I fixed it, beside that I moved Dockerfile into /tools so we can add more tools down the road